packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -50,7 +57,9 @@ function TagGroup<T extends object>(props: SpectrumTagGroupProps<T>, ref: DOMRef
       ? new TagKeyboardDelegate(new ListCollection([...state.collection].slice(0, tagState.visibleTagCount)), direction)
       : new TagKeyboardDelegate(new ListCollection([...state.collection]), direction)
   ), [direction, isCollapsed, state.collection, tagState.visibleTagCount]) as TagKeyboardDelegate<T>;
-  let {tagGroupProps} = useTagGroup({...props, keyboardDelegate}, state, domRef);
+  // Remove onAction from props so it doesn't make it into useGridList.
+  delete props.onAction;
+  let {tagGroupProps, tagGroupActionsProps} = useTagGroup({...props, keyboardDelegate}, state, domRef);
```

Comment:
What can't we use `otherProps` instead of `props` with `onAction` deleted? I looked at useTagGroup and useGridList and didn't see any of the props not in otherProps being used.

Reply:
We could, but usually we've just passed all the props down. There just happened to be a name collision with this one.

I think, as @snowystinger mentioned in his comment above, we could consider a different name for this prop as to not confuse people (and mitigate issues if we end up ever supporting individual tag actions). Maybe `onGroupAction` and `groupActionLabel`?